### PR TITLE
User profile details retrieved for single user.

### PR DESCRIPTION
**Pull request by heymonicakay**
#### Changes Made
1. Modified file `user.py` in `views` to include `retrieve` functionality to get single user.
​1. Modified file pull request template to simplify and streamline template.
1. Removed duplicate import from file `urls.py`
1. Modified fixtures `rareusers.json` and `users.json` to include an active admin, an active author, and an inactive author in user db.
1. Included descriptive doc strings on unmapped properties and added `is_current_user` property in `rareuser.py` model.
1. Added doc strings to `__init__.py` and `auth.py` in views.
1. Imported `CategorySerializer` to `post.py` in views & added `id` to fields on `PostRareUserSerializer` 

#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout mp-user-profile
```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
4. Nuke your db to access updated database
```
./nuke.sh
```
5. Test app functionality.
> Create a GET request in Postman with URL `http://localhost:8000/users/2` and auth token `fa2eba9be8282d595c997ee5cd49f2ed31f65bed` (for user 1)
> Expected return:
```
{
    "id": 2,
    "bio": "World renowned book enthusiast, instantgram influencer, and ticktockticktockticktock kween Theo Daly began building her empire on the popular social media platform SnapCrackleChat in 2001. With a net worth of $8.1 bazillion, Forbey Parker ranked her somewhere in the middle of their list of the world’s kajillionaires.",
    "is_staff": false,
    "is_active": true,
    "full_name": "Theo Daly",
    "profile_image_url": null,
    "is_current_user": false,
    "username": "theocanread",
    "email": "theo@daly.com",
    "date_joined": "2020-08-28T14:51:39.989000Z"
}
```
> Create a GET request in Postman with URL `http://localhost:8000/users/2` and auth token `fa2eba9be8282d595c997ee5cd49f2ed31f65bed` (for user 1)
> Expected return:
```
{
    "id": 1,
    "bio": "Internationally acclaimed scuba instructor, radio dj personality, and mid-century modern sofa collector, Mary Mogul is often referred to as 'Awesome', 'Super Fantastic', and 'Great'. Her talk show 'Mary The Mogul' ran from 2019 to 2020 and made history by becoming the most 'wow' program of its kind ever to be aired, and the most 'woah' program of its kind ever to be cancelled. With a net worth of $3.1 gazillion, Forbey Parker ranked her at the top of their list of the world’s kajillionaires.",
    "is_staff": true,
    "is_active": true,
    "full_name": "Mary Mogul",
    "profile_image_url": null,
    "is_current_user": true,
    "username": "marymogul",
    "email": "mary@mogul.com",
    "date_joined": "2020-08-28T14:51:39.989000Z"
}
```
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.

### CODE REVIEW

File: rareapi/views/user.py [hunk 1]
```diff
@@ -20,23 +20,48 @@ def list(self, request):
...
+        """Handles GET requests to users resource for single User
+        Written for User Profile View
+        Returns:
+            Response -- JSON serielized rareuser instance
+        """
+        try:
+            rareuser = RareUser.objects.get(pk=pk)
+
+            rareuser.is_current_user = None
+            current_rareuser = RareUser.objects.get(user=request.auth.user)
```
Comment (Raparker88) [inline rareapi/views/user.py:34]: I don't think you need this line

**Review by Raparker88 — COMMENTED** [review]
You'll have to import the post serializer since you deleted the hard code in user.py view. Since the custom action posts uses the post serializer. This is why clicking my posts while running the react app won't work with your branch at the moment.

**Review by atphy — APPROVED** [review]
looks solid with the changes @Raparker88  mentioned!

**Comment by heymonicakay** [comment]
@Raparker88 @atphy thank you for catching this! I fixed the imports in 'user.py', and '/users/posts' view is working again.

**Review by Raparker88 — APPROVED** [review]

**Pull request merged**